lib/accessory.js

```diff
@@ -54,15 +64,19 @@ class Accessory {
   }
 
   setOutlet (uuid, powered, done) {
-    this.remoteOutlet(uuid).update({desired_state: {powered}}, () => {
+	  var that = this;
```

Comment:
all good, you can learn more about [arrow functions here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) (fairly similar to coffee script arrow function indeed).